### PR TITLE
Approved on 2024-08-05 - Open Source in Business

### PR DESCRIPTION
**Pull request by anajsana**

### CODE REVIEW

**Comment by lhawthorn** [comment]
LGTM, I am supportive of the proposal and want to participate in crafting the document.

**Comment by BrittanyIstenes** [comment]
+1 - looks good to me as well

**Comment by justaugustus** [comment]
Since the automatic resolution [vote trigger](https://github.com/todogroup/governance/pull/337) is not merged yet (sorry!):
/vote

**Comment by justaugustus** [comment]
@todogroup/steering-committee — I'm retriggering this, since the vote timed out:
/vote

File: resolutions/proposed/2024-28.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+Resolution YYYY-NN
+Status: Proposed on 2024-06-24
+Subject: New Focuss Group - Open Source in Business Collaborative Whitepaper / Guide
```
Comment (ashleywolf) [inline resolutions/proposed/2024-28.md:3]: Focus

**Comment by justaugustus** [comment]
Between the results of the [previous vote](https://github.com/todogroup/governance/pull/336#issuecomment-2252993871):

Screenshot 2024-08-04 at 19 55 37

And the [current vote](https://github.com/todogroup/governance/pull/336#issuecomment-2267344545):

Screenshot 2024-08-04 at 19 55 56

We have 5 @todogroup/steering-committee votes, so this resolution passes.
/cancel vote

**Comment by justaugustus** [comment]
take 3:
/cancel-vote

File: resolutions/proposed/2024-28.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+That the committee {approves/declines the proposed action}.
+Vote Results
+
+    Annania Melaku - {approve/decline/abstain/absent}
+    Ashley Wolf - {approve/decline/abstain/absent}
+    Brittany Istenes - {approve/decline/abstain/absent}
+    Georg Kunz - {approve/decline/abstain/absent}
+    Leslie Hawthorn - {approve/decline/abstain/absent}
+    Nik Peters - {approve/decline/abstain/absent}
+    Stephen Augustus - {approve/decline/abstain/absent}
```
Comment (justaugustus) [inline resolutions/proposed/2024-28.md:27]:
```suggestion
    Annania Melaku - approve
    Ashley Wolf - approve
    Brittany Istenes - absent
    Georg Kunz - approve
    Leslie Hawthorn - approve
    Nik Peters - absent
    Stephen Augustus - approve
```

ref: https://github.com/todogroup/governance/pull/336#issuecomment-2268082101

File: resolutions/proposed/2024-28.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
+Resolution YYYY-NN
+Status: Proposed on 2024-06-24
```
Comment (justaugustus) [inline resolutions/proposed/2024-28.md:2]:
```suggestion
Status: Accepted
Proposed date: 2024-06-24
```

File: resolutions/proposed/2024-28.md (hunk 1 above)
Comment (justaugustus) [reply resolutions/proposed/2024-28.md:3]:
```suggestion
Subject: New Focus Group - Open Source in Business Collaborative Whitepaper / Guide
```

**Review by justaugustus — CHANGES_REQUESTED** [review]
Acknowledging the passing of the resolution and marking status as "Accepted".

File: resolutions/proposed/2024-28.md
Comment (justaugustus) [inline resolutions/proposed/2024-28.md:1]: Rename to `resolutions/approved/2024-06-24.md`

File: resolutions/proposed/2024-28.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
+Resolution YYYY-NN
+Status: Accepted
+Proposed date: 2024-06-24
+Subject: New Focus Group - Open Source in Business Collaborative Whitepaper / Guide
+
+ ## Proposal
+ 
+During the Touchpoint call on Thursday, May 16th, participants presented the idea of building a collaborative project focused on open source in business. The proposal can be found [here](https://github.com/todogroup/ospology/discussions/481).
+
+The following month, during the Touchpoint call on Thursday, June 27th, the community reviewed the interest shared (see comments bellow [here](https://github.com/todogroup/ospology/discussions/481)) and started to draft a list of challenges and
```
Comment (justaugustus) [inline resolutions/proposed/2024-28.md:10]:
```suggestion
The following month, during the Touchpoint call on Thursday, June 27th, the community reviewed the interest shared (see comments [here](https://github.com/todogroup/ospology/discussions/481)) and started to draft a list of challenges and
```

File: resolutions/proposed/2024-28.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+agreed to define an outline to enable the group to begin creating content.
+
+The request is for the SC to review this proposal to approve the creation of a new repository under TODO GH org to work collaboratively on the outline of this guide.
+
+Resolved:
+
+WHEREAS, the charter of the TODO Group, dated YYYY-MM-DD, empowers the Steering Committee to {category of action};
```
Comment (justaugustus) [inline resolutions/proposed/2024-28.md:17]: Needs update

File: resolutions/proposed/2024-28.md [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+
+    Annania Melaku - approve
+    Ashley Wolf - approve
+    Brittany Istenes - absent
+    Georg Kunz - approve
+    Leslie Hawthorn - approve
+    Nik Peters - absent
+    Stephen Augustus - approve
+
+Rationale
```
Comment (justaugustus) [inline resolutions/proposed/2024-28.md:30]: Needs update (and heading?)

File: resolutions/proposed/2024-28.md [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+agreed to define an outline to enable the group to begin creating content.
+
+The request is for the SC to review this proposal to approve the creation of a new repository under TODO GH org to work collaboratively on the outline of this guide.
+
+Resolved:
+
+WHEREAS, the charter of the TODO Group, dated YYYY-MM-DD, empowers the Steering Committee to {category of action};
+
+That the committee {approves/declines the proposed action}.
+Vote Results
```
Comment (justaugustus) [inline resolutions/proposed/2024-28.md:20]: Needs heading?

File: resolutions/proposed/2024-28.md [hunk 8]
```diff
@@ -0,0 +1,30 @@
...
+The following month, during the Touchpoint call on Thursday, June 27th, the community reviewed the interest shared (see comments bellow [here](https://github.com/todogroup/ospology/discussions/481)) and started to draft a list of challenges and
+agreed to define an outline to enable the group to begin creating content.
+
+The request is for the SC to review this proposal to approve the creation of a new repository under TODO GH org to work collaboratively on the outline of this guide.
+
+Resolved:
+
+WHEREAS, the charter of the TODO Group, dated YYYY-MM-DD, empowers the Steering Committee to {category of action};
+
+That the committee {approves/declines the proposed action}.
```
Comment (justaugustus) [inline resolutions/proposed/2024-28.md:19]:
```suggestion
That the committee approves the proposed action.
```

File: resolutions/proposed/2024-28.md [hunk 9]
```diff
@@ -0,0 +1,30 @@
...
+Subject: New Focus Group - Open Source in Business Collaborative Whitepaper / Guide
+
+ ## Proposal
+ 
+During the Touchpoint call on Thursday, May 16th, participants presented the idea of building a collaborative project focused on open source in business. The proposal can be found [here](https://github.com/todogroup/ospology/discussions/481).
+
+The following month, during the Touchpoint call on Thursday, June 27th, the community reviewed the interest shared (see comments bellow [here](https://github.com/todogroup/ospology/discussions/481)) and started to draft a list of challenges and
+agreed to define an outline to enable the group to begin creating content.
+
+The request is for the SC to review this proposal to approve the creation of a new repository under TODO GH org to work collaboratively on the outline of this guide.
```
Comment (justaugustus) [inline resolutions/proposed/2024-28.md:13]:
```suggestion
The request is for the Steering Committee to review this proposal to approve the creation of a new repository under TODO Group GitHub organization to work collaboratively on the outline of this guide.
```

**Review by justaugustus — CHANGES_REQUESTED** [review]
@anajsana — I've covered some initial cleanups in https://github.com/todogroup/governance/pull/336/commits/e6e3f4ec35f0c72ed0a1da8c697152f40cb3e513 and left some suggestions/needs in this review. Over to you :)

**Comment by anajsana** [comment]
> Between the results of the [previous vote](https://github.com/todogroup/governance/pull/336#issuecomment-2252993871):
> Screenshot 2024-08-04 at 19 55 37
> 
> And the [current vote](https://github.com/todogroup/governance/pull/336#issuecomment-2267344545):
> Screenshot 2024-08-04 at 19 55 56
> 
> We have 5 @todogroup/steering-committee votes, so this resolution passes. /cancel vote

Realized that there was final consensus that this was approved. Thanks all!

File: resolutions/proposed/2024-28.md [hunk 10]
```diff
@@ -0,0 +1,30 @@
...
+During the Touchpoint call on Thursday, May 16th, participants presented the idea of building a collaborative project focused on open source in business. The proposal can be found [here](https://github.com/todogroup/ospology/discussions/481).
+
+The following month, during the Touchpoint call on Thursday, June 27th, the community reviewed the interest shared (see comments [here](https://github.com/todogroup/ospology/discussions/481)) and started to draft a list of challenges and
+agreed to define an outline to enable the group to begin creating content.
+
+The request is for the Steering Committee to review this proposal to approve the creation of a new repository under TODO Group GitHub organization to work collaboratively on the outline of this guide.
+
+Resolved:
+
+WHEREAS, the charter of the TODO Group, dated YYYY-MM-DD, empowers the Steering Committee to {category of action};
```
Comment (anajsana) [inline resolutions/proposed/2024-28.md:17]:
```suggestion
The Steering Committee approves the creation of a new repository under TODO Group GitHub organization to work collaboratively on the outline of this guide;
```

File: resolutions/proposed/2024-28.md [hunk 11]
```diff
@@ -0,0 +1,30 @@
...
+
+That the committee approves the proposed action.
```
Comment (anajsana) [inline resolutions/proposed/2024-28.md:19]:
```suggestion
That the committee approves the creation of a new repository under TODO Group GitHub organization to work collaboratively on the outline of this guide.
```

File: resolutions/proposed/2024-28.md [hunk 12]
```diff
@@ -0,0 +1,30 @@
...
+Vote Results
```
Comment (anajsana) [inline resolutions/proposed/2024-28.md:20]:
```suggestion
## Vote Results
```

File: resolutions/proposed/2024-28.md [hunk 13]
```diff
@@ -0,0 +1,30 @@
+Resolution YYYY-NN
```
Comment (anajsana) [inline resolutions/proposed/2024-28.md:1]:
```suggestion
Resolution 2024-08
```

**Pull request merged**